### DISCUSSION TITLE
Asking for help to dive into

### DISCUSSION THREAD

**Discussion by olehbozhok** [JanssenProject/jans#9224] [Q&A]
I watched couple videos from https://gluu.org/agama-project-of-the-week/

so as I understand, I need to up server

https://github.com/JanssenProject/jans/tree/main/docker-jans-monolith

so I made it

(I run on windows, but if it need can reboot on linux mint)
so I started 
`bash up.sh`
 and in docker file changed mapping 80 port to 8083

and started to configure service inside docker container
docker compose -f jans-mysql-compose.yml exec jans
started bash with command bash

installed python
 apt install python3

and on this moment can't understand what exactly I need to configure

runned 
```
python3 /opt/jans/jans-cli/config-cli.py --output-access-token
```

it returned nothing....

What I need to setup next to start explore this "creature"?

**Comment by nynymike** [comment]
@misba7  is windows supported for docker monolith?

**Reply by misba7** [reply]
Linux containers requires Linux kernel, and Windows containers requires Windows kernel.

However, we can still run our Ubuntu-based container on a Windows machine using a compatibility layer like WSL 2, since it runs a real Linux kernel within Windows.
 
You can set up Docker Desktop for Windows to use WSL 2 as the backend
https://docs.docker.com/desktop/wsl/

**Comment by misba7** [comment]
You don't need to install python inside the container. It's already installed.

`docker exec -it <jans-container-id> python3 --version`

And you should have a working installation out of the box. 
To double-check, map the Jans-hostname(defaults to demoexample.jans.io) to the VM IP in /etc/hosts file, and you can hit `demoexample.jans.io/.well-known/openid-configuration` in the browser.

And the cli is located inside the container as you pointed at `/opt/jans/jans-cli/config-cli.py`

We also have a full section named "Configuration Guide" in the [docs](https://docs.jans.io/v1.1.4/admin/config-guide/config-tools/jans-cli/#on-the-janssen-server)

**Comment by olehbozhok** [comment]
The solution is
call in container
```
/opt/jans/jans-cli/config-cli.py --client-id 9876baac-de39-4c23-8a78-674b59df8c09 --client-secret 1t5Fin3#security -auth
```
and after 
```
 /opt/jans/jans-cli/config-cli.py   --output-access-token
 ```

**Comment by NateNjuguna** [comment]
I have recently been on this same path & wrote up a guide to set up Janssen on Windows with Kubernetes & Helm [here](https://medium.com/@natenjuguna/install-janssen-server-on-windows-with-kubernetes-helm-b0c829c31952)
